CMakeLists.txt

```diff
@@ -42,13 +42,16 @@ install(DIRECTORY include/
   DESTINATION include
 )
 
-install(TARGETS DepthImageToLaserScan DepthImageToLaserScanROS depthimage_to_laserscan_node
-  EXPORT export_depthimage_to_laserscan_node
```

Comment:
@clalancette I was not sure if this was needed, maybe you have a better idea of what this was used for. I'm happy to add it back if it is required